README.md

```diff
@@ -2,24 +2,37 @@
 
 An interpreted programming language that focuses on simplicity and usability. The main goal of Night is to design an intuitive and easy to learn language.
 
-It's still in early development, and there's a lot to get done. The goal here is to eventually be able to code Night using Night.
+It's still in early development, and there's a lot to get done. The goal here is to eventually be able to bootstrap Night.
 
 ---
 
-### Getting Started with Night
+## Getting Started with Night
 
 Quick and easy installating and setup can be found on the [Night website](https://night-website.dynamicsquid.repl.co/index.html). There you'll also find instructions on how to use *dusk*, the package manager.
 
 You can also build Night from source if you'd like.
 
-### Building from Source
+## Building from Source
 
 1. Install *g++* or *clang++*
 
 2. Clone this repository
 
+With *git*:
+
 ```
 git clone https://github.com/DynamicSquid/night.git
+```
+
+With *gh*: the GitHub CLI:
```

Comment:
Bro who even uses github cli? Like I know it's cool and everything but just use git

Reply:
Well the rustlings docs supply instructions for both (i think)

Reply:
Well, a better version of github cli already exists, it's called git

Reply:
Actually it's the other way round but uh RUSTLINGS SUPPLIES INSTRUCTIONS FOR BOTH

Reply:
Actually it doesn't but I can't remember where it did